be/active_record.c

```diff
@@ -51,7 +51,7 @@ M0_BE_LIST_DESCR_DEFINE(rds, "list of active_record in ar_domain_subsystem",
 			M0_BE_ACT_REC_DOM_SUB_MAGIC, M0_BE_ACT_REC_DOM_SUB_MAGIC);
 M0_BE_LIST_DEFINE(rds, static, struct m0_be_active_record);
 
-
+
```

Comment:
Fixed

Reply:
I see now that you removed space at the end of line. It's OK, but since there are a lot of such changes in this patch, they produce a lot of odd noise and distraction. So I'd recommend cutting out all these changes into a separate PR against the main branch, so they could be reviewed and landed sooner and separately.

Reply:
There is no need to create separate PR for style changes as this PR hold all the style changes, and moreover there are few style changes I suppose